### DISCUSSION TITLE
Problem with plugin "netbox.netbox.nb_inventory"

### DISCUSSION THREAD

**Discussion by Cob87** [netbox-community/netbox#12952] [General]
Hello, 

I have a problem when I use the "netbox.netbox.nb_inventory" plugin since the upgrade to v3.5.0.
if I understand the error message correctly, it seems that the plugin is trying to contact "api/doc" but that it no longer exists.

![inventory_conf](https://github.com/netbox-community/netbox/assets/45387621/b10cae10-7452-4a06-b21f-b6d8c47debbc)
![error_Capture](https://github.com/netbox-community/netbox/assets/45387621/e6b63d24-80ca-4dae-9b3d-357d0e9008d7)
![error_parsing](https://github.com/netbox-community/netbox/assets/45387621/369a564d-ea9d-44a4-8a0f-7676884e3c9d)

ansible-playbook [core 2.13.7]
python version = 3.8.10 (default, May 26 2023, 14:05:08) [GCC 9.4.0]
jinja version = 3.1.2
netbox.netbox 3.9.0
new netbox version: v3.5.0
old netbox version:  v3.0.9

Do you have any information about that?

Thanks

**Comment by rodvand** [comment]
Update your ansible collection.

**Reply by Cob87** [reply]
Thank you! 
that's what I had done. But after checking, I realized that the netbox collection was still in 3.9.0.
After a few actions, it now works using version 3.13.0

Have a great day.
